apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -159,9 +159,8 @@ class OppgaveService(
     ) {
         val hentetOppgave =
             oppgaveDao.hentOppgave(oppgaveId) ?: throw OppgaveIkkeFunnet(oppgaveId)
-        sikreAktivOppgaveOgTildeltSaksbehandler(hentetOppgave) {
-            oppgaveDao.oppdaterStatusOgMerknad(oppgaveId, merknad, status)
-        }
+        sikreAtOppgaveIkkeErAvsluttet(hentetOppgave)
```

Comment:
Tror kanskje vi vil være litt strengere og at kun dette gjelder gjenoppretta?

Reply:
For vanlige oppgaver blir det kanskje litt rart om de kan endre til andre ting uten at saksbehandler er tildelt

Reply:
Åja my bad det her gjelder ikke gjenoppretta oppgaver ja

Reply:
> Åja my bad det her gjelder ikke gjenoppretta oppgaver ja. Anywho føler jeg det burde vært snevret inn til den jobben?

Hmm, trur ikkje eg synes det er forskjell på om oppgåva blir tatt av vent automatisk av systemet eller manuelt av ein saksbehandlar.